test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
với sample app, toàn bộ thư mục test bạn đưa vào .gitignore nhé, và dùng câu lệnh sau để remove nó khỏi cache
`git rm -r --cached test/`